src/app/story/storysidebar/storysidebar.component.ts

```diff
@@ -45,4 +43,29 @@ export class StorysidebarComponent implements OnInit, OnChanges {
     }
     return false;
   }
+
+  filterProperties(node: any): any {
```

Comment:
Just out of curiosity, can you please tell me why the Pipe was converted into a method?